server/perkeepd/ui/index.js

```diff
@@ -80,6 +81,7 @@ cam.IndexPage = React.createClass({
 		cam.BlobItemInstapaperContent.getHandler,
 		cam.BlobItemImageContent.getHandler,
 		cam.BlobItemVideoContent.getHandler,
+		cam.BlobItemAudioContent.getHandler,
```

Comment:
Just move the audio handler above the video handler, and I think that should be enough to give the audio handler priority for oggs.